data/moves/descriptions.asm

```diff
@@ -478,6 +478,9 @@ WillOWispDescription:
 
 StunSporeDescription:
 ThunderWaveDescription:
+	db   "A move that"
+	next "causes paralysis.@"
+
 GlareDescription:
```

Comment:
Shouldn't Glare also be updated?

Reply:
Sorry it took me forever to see this. Yes, Glare should probably have the same description as Thunder Wave saying it always paralyzes (it it hits; Glare only has 75% accuracy.

Reply:
Actually, based on how Hypnosis's description says it "MAY cause the foe to fall asleep," while Spore says it "PUTS the foe to sleep," I assume moves with 100% accuracy are supposed to say they "may" cause a status effect.

Reply:
I made a new pull request that includes more changes.